content/_releases/25/50.md

```diff
@@ -0,0 +1,69 @@
+---
+title: Product Update - v25.06
+description: Details of the product update v25.06 from the 6th of February 2025.
+layout: rel
+version: "25.06"
+section: Product Updates
+category: product-updates
+releaseDate: 2024-02-06
+redirect_from:
+  - /releases/25.06.html
+---
+
+## New Features
+### New endpoint for retrieving sorted Workspaces list
+Enhancing workspace management capabilities with the introduction of a new API endpoint `GET /workspaces/get_workspaces_all_list` designed to deliver a sorted list of available workspaces based on user permissions within the corresponding contract. 
+This endpoint enables users to retrieve a comprehensive view of all workspaces they are authorized to access, streamlining workspace navigation and organization.
+
+### Recipes API and UI enhancement: Search functionality added to v2/recipes endpoint
+Implemented a search functionality enhancement for the `GET v2/recipes` endpoint, allowing users to search for Recipes based on a specific word or phrase contained within the recipe title.
+By incorporating this search functionality into both the API and user interface, users can easily locate and access recipes that match their search terms, streamlining the recipe discovery process and improving overall usability within the system.
+
+## Fixed Bugs
+*   **FIXED** the issue of retried messages getting stuck in the retry queue and fixed the rendering of the Queues page for retry flows.
```

Comment:
we didn't fiw stuck messages. we just afix rendering the queues page for retry flow https://github.com/elasticio/elasticio/issues/7576#issuecomment-2345854730